daemon/algod/api/server/v2/utils.go

```diff
@@ -39,12 +40,18 @@ import (
 	"github.com/algorand/go-algorand/logging"
 	"github.com/algorand/go-algorand/node"
 	"github.com/algorand/go-algorand/protocol"
+	"github.com/algorand/go-algorand/serr"
```

Comment:
maybe move to data/serr or components/serr? Looks too small for a separate top-level pkg

Reply:
how about `basics`? The type would become `basics.SError` instead of `serr.Error`.

`data` seems to be about algorand specific structures, and components only has `mocks` which makes it seem like it's for tests?

This type should certainly never have Algorand dependencies, so it should be safe to put in `basics` with respect to cyclic imports.

Reply:
Oh, `basics` _is_ under `data`.  I think we agree then, I'll move it in there.